core/src/main/java/tc/oc/pgm/command/MapPoolCommand.java

```diff
@@ -275,7 +275,11 @@ public static void skip(
 
   @Command(aliases = "votenext", desc = "Vote for the next map", usage = "map")
   public static void voteNext(
-      MatchPlayer player, CommandSender sender, MapOrder mapOrder, @Text MapInfo map)
+      MatchPlayer player,
+      CommandSender sender,
+      MapOrder mapOrder,
+      @Switch('f') boolean force,
```

Comment:
I don't think this needs to be a parameter for the command
```suggestion
```

Reply:
If I do `votenext The Nile` I don't want the book to popup.

Reply:
Without seeing the book, you can't even know `The Nile` is an option, neither will you know if it's been voted for or against. The command is an implementation detail for how the book works, never intended to be used standalone. If you wish to make it a command make it so that skipping the -f flag will display voting messages in chat that can replace the functionality of the book.